src/worker.js

```diff
@@ -27,7 +27,7 @@ onmessage = initMessage => {
       eval(initMessage.data.startScript);
     }
 
-    const _normalizeUrl = src => new URL(src, initMessage.data.baseUrl).href;
+    const _normalizeUrl = src => new URL(src, new URL(initMessage.data.baseUrl).origin).href;
```

Comment:
Does this mean the correct behavior of worker src is to be not relative to script path, but just the origin?

I’m trying to find the reference that says that’s correct but all I could find was https://html.spec.whatwg.org/multipage/workers.html#dom-worker

Reply:
All I know is that without this PR, I had to change the script url from “examples/js/ammo.js” to “js/ammo.js” before it would work in exokit. That change of course broke it in chrome.

I don’t know if origin relative urls are correct, but it seems hard to think of other possibilities.

That’s not a very satisfying answer, but it’s pretty strong circumstantial evidence.

Reply:
Thanks for the research, looks like muddy waters anyway. https://stackoverflow.com/a/12417265

Fine to just do what Chrome does.